pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  test: ^1.5.1
```

Comment:
If I'm not mistaken there are far newer version available
https://pub.dev/packages/test/changelog
`1.21.6`

Reply:
^ will update it to latest minor version automatically

Reply:
Yes, but it works only for major versions with use of this command
`flutter pub upgrade --major-versions`

Automatic updates are done only for non-braking changes.

My comment here is a suggestion of manual upgrade to latest version and add code modifications after upgrade, if required.

Reply:
fixed